src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -210,12 +210,6 @@
             },
             "description": "Layout of panes in the RStudio workbench."
         },
-        "enable_additional_columns": {
-           "type": "boolean",
-           "default": false,
```

Comment:
What do you think about leaving this option in and just changing the default to `true`? The reason I suggest this is that it's come up at least once in conversation that in some environments we might actually want to constrain the UI (e.g. in a tutorial mode where we need to preserve the "only one file is open" invariant). We could use the option to do this since you've already done the wiring, though we might want to rename it to something more specific.

Reply:
Done! I added "Allow Source Columns" which defaults to true.

Reply:
Anything to worry about if a user has added source columns, then switches off this preference? Do existing source columns get closed or do they continue to function, that sort of thing?

Reply:
The columns will persist until the user refreshes the browser, then any open tabs are moved to the main source column. The `Remove Column` button in Global Options is still enabled after the option is turned off so the user could go there to remove columns. Removing them as soon as the option is set to Off seems jarring.

Reply:
Not suggesting we remove them when option is turned off, was just curious how it behaves.